frontend/src/views/setup/Setup.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+  <v-container class="mt-7">
+    <v-row>
+      <v-col>
+        <img src="@/assets/logo_v2_full.png" alt="logo" width="480">
+      </v-col>
+    </v-row>
+    <v-row v-if="!done">
+      <v-col>
+        <v-card>
+          <v-card-title>Request a license</v-card-title>
+          <v-card-text>
+            <p>
+              Giskard server requires a license. A <span class="font-weight-bold">free</span> license can be obtained by
+              registered using the form below. The license will be sent by email.
+            </p>
+            <p>If you already have one, you can upload it on the right hand side of this page.</p>
+            <v-form>
+              <v-text-field label="First name" v-model="firstName"></v-text-field>
+              <v-text-field label="Last name" v-model="lastName"></v-text-field>
+              <v-text-field label="Email" v-model="email"></v-text-field>
+              <v-text-field label="Company name" v-model="companyName"></v-text-field>
+              <v-btn :loading="loading" text @click="submit">Submit</v-btn>
+            </v-form>
+          </v-card-text>
+        </v-card>
+      </v-col>
+      <v-divider vertical></v-divider>
+      <v-col>
+        <v-card>
+          <v-card-title>Use existing license</v-card-title>
+          <v-card-text>
+            If you already have a license, you can upload your license file by pressing the button below.
+          </v-card-text>
+          <v-card-actions class="ml-2 mb-2">
+            <v-btn small color="primary" @click="openFileInput">Upload license file</v-btn>
+            <input type="file" ref="fileInput" style="display: none;" @change="onFileUpdate"/>
+          </v-card-actions>
+        </v-card>
+      </v-col>
+    </v-row>
+    <v-row v-if="done">
+      <v-col>
+        <v-card>
+          <v-card-title>Thank you for installing Giskard</v-card-title>
```

Comment:
I added some things to ensure it doesn't happen, but it didn't happen very consistently for me. If you manage to make it happen again with my fixes now I'd love to know, because now I can't manage to reproduce it.